ci-operator/step-registry/ipi/install/nutanix/monitoringpvc/ipi-install-nutanix-monitoringpvc-commands.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
There's nothing Nutanix-specific here. Lets rename it into "ipi-install-post-monitoringpvc" to show that its expected to run in post-install phase